src/ducks/apps/components/ApplicationPage/Details.jsx

```diff
@@ -59,7 +61,12 @@ export class Details extends Component {
     this.onUpdateChannel = this.onUpdateChannel.bind(this)
   }
 
-  toggleDisplayMore(type) {
+  onShowPermissions () {
+    const { app } = this.props
+    this.props.history.push(`/myapps/${app.slug}/permissions`)
```

Comment:
You have to use the `parent` props here because you can be in the `myapps` route as in the `discover` route at this point.

Reply:
Well spotted !